components/viz/service/display/gl_renderer.cc

```diff
@@ -600,7 +600,9 @@ void GLRenderer::DoDrawQuad(const DrawQuad* quad,
       DrawSolidColorQuad(SolidColorDrawQuad::MaterialCast(quad), clip_region);
       break;
     case DrawQuad::STREAM_VIDEO_CONTENT:
+#if !defined(CASTANETS)
       DrawStreamVideoQuad(StreamVideoDrawQuad::MaterialCast(quad), clip_region);
```

Comment:
Why did you ignore video quad rendering?

Reply:
Currently, StreamVideoQuad and YUV quads are not encountered in the supported scenarios.
Also it needs some verification to make the changes for these quads.
So I have added a TODO for now. Once these quads are supported/needed we will add a fixup.
Should that be fine ?